security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through policy management systems on macOS and Windows.
```

Comment:
Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed.

policy is word used for Windows configurations, for macOS the correct wording is configuration profiles. 
So, instead of word policies, I recommend using "configurations" in this sentence.